src/index.tsx

```diff
@@ -536,7 +536,7 @@ class DraggableFlatList<T> extends React.Component<Props<T>, State> {
           : !flatListNode
           ? "no flatlist node"
           : "invalid ref";
-        console.log(`## can't measure ${key} reason: ${reason}`);
+        console.log(`## can't measure ${key} reason: ${reason}, the parent node should specify the dimension, example: if you're using <View> set it to <View style={{flex:1}}`);
```

Comment:
I don't think this is _always_ the reason the measurement fails -- can you add the string to the `reason` above?
I'm guessing it's the "invalid ref" case?